src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -235,10 +238,10 @@ const useCountStepTable = ({
           <Component
             disabled={isFieldEditable}
             type={type}
-            value={value}
+            value={showNoLotNumber ? translate('react.cycleCount.noLot.label', 'NO LOT') : value}
```

Comment:
Is this the correct place to handle this? @kchelstowski @alannadolny, this feels odd. If there will be any conditionally displayed value on the other column, then everything needs to be handled here? Asking you guys for advise, because you have better knowledge about these components/hooks.